docs/source/algorithms/ScaleInstrumentComponent-v1.rst

```diff
@@ -0,0 +1,263 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This scales the detecors and pixels in a component. This can effectively expand or contract the overall dimensions
+of the component by resizing its detector pixels, without moving its geometrical center. You can choose to scale the component
+without scaling the size of the detector pixels. This would allow you to expand or contract the detectors without changing the
```

Comment:
```suggestion
without scaling the size of the detector pixels. This would allow you to expand or contract the detector positions without changing the
```